backstop/tests/carousel.html

```diff
@@ -0,0 +1,270 @@
+    <hr />
+
+    <h2>Default</h2>
+    <section id="demo-default">
+
+      <div
+        class="iui-carousel demo-carousel-width"
+        aria-roledescription="carousel"
+        aria-label="Introduction to iTwinUI"
+        aria-live="polite"
+        aria-atomic="true"
+      >
+        <ol
+          class="iui-carousel-slider demo-carousel-height"
+          id="myCarousel-items"
+        >
+
+          <!-- Slide with custom interior content. -->
+          <li
+            class="iui-carousel-slider-item demo-carousel-height iui-current"
+            data-slide-id="0"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="1 of 3"
+            aria-hidden="false"
+          >
+            <div class="demo-columns">
+              <div class="demo-column demo-left">
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+                tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation
+                ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+                voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+                proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+              </div>
+              <div class="demo-column demo-right"></div>
+            </div>
+          </li>
+
+          <!-- Slide with full frame video embed. -->
+          <li
+            class="iui-carousel-slider-item demo-carousel-height iui-next"
+            data-slide-id="1"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="2 of 3"
+            aria-hidden="true"
+          >
+            <iframe
+              width="100%"
+              height="100%"
+              src="https://www.youtube.com/embed/DT59xdKNbA8"
+              title="YouTube video player"
+              frameborder="0"
+              allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+              allowfullscreen
+            ></iframe>
+          </li>
+
+          <!-- Slide with image and title. -->
+          <li
+            class="iui-carousel-slider-item demo-carousel-height iui-next"
+            data-slide-id="2"
+            role="group"
+            aria-roledescription="slide"
+            aria-label="3 of 3"
+            aria-hidden="true"
+          >
+            <div
+              class="iui-carousel-slider-item-image"
+              style="background-image: url('../assets/image-test.jpg');"
+            ></div>
+            <div class="iui-carousel-slider-item-title">Bentley System headquarters</div>
+          </li>
+
+        </ol>
+
+        <nav class="iui-carousel-navigation">
+          <button
+            class="iui-button iui-borderless iui-small iui-carousel-navigation-button iui-carousel-navigation-previous"
```

Comment:
So specific nagivation button classes are not used anymore, can be removed.

Reply:
Was able to remove `.iui-carousel-navigation-previous` & `.iui-carousel-navigation-next`.